pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -130,8 +131,9 @@ func (pmpt *Action) Execute(ssn *framework.Session) {
 					}
 					// Preempt other jobs within queue
 					return job.Queue == preemptorJob.Queue && preemptor.Job != task.Job
-				}, ph); preempted {
-					assigned = true
+				}, ph)
+				if err != nil {
+					klog.V(3).Infof("Failed to preempt Task , err: %s", err)
```

Comment:
We'd better add current task name info in log

Reply:
make sense to me, done